src/mgmt/services/manager/project/api/models.py

```diff
@@ -70,13 +71,18 @@ class Node(db.Model):
     __tablename__ = 'nodes'
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    zgc_id = db.Column(db.String(64), db.ForeignKey('zgcs.zgc_id'), nullable=False)
+    zgc_id = db.Column(db.String(64), db.ForeignKey(
+        'zgcs.zgc_id'), nullable=False)
```

Comment:
Can we break at , instead of middle of a field?

Reply:
Sure, done.